app/Domain/Integrations/UdbOrganizer.php

```diff
@@ -11,7 +11,18 @@
     public function __construct(
         public UuidInterface $id,
         public UuidInterface $integrationId,
-        public string $organizerId
+        public string $organizerId,
+        public UdbOrganizerStatus $status = UdbOrganizerStatus::Pending,
```

Comment:
This shouldn't be part of the default constructor, see `Integration::withKeyVisibility`

Reply:
I am happy to change it but can you explain a little bit why?
I considered your approach, but I went with a default value because this lowers the amount of files/code I have to change, because this object is created in 15 different places.

Reply:
After slack discussion, we agreed on making it required.